.goreleaser.yml

```diff
@@ -36,10 +36,32 @@ dockers:
 - goos: linux
   goarch: amd64
   image_templates:
-  - "percona/mongodb_exporter:{{.Major}}.{{.Minor}}"
-  - "percona/mongodb_exporter:{{.Version}}"
+  - "percona/mongodb_exporter:{{.Major}}.{{.Minor}}-amd64"
+  - "percona/mongodb_exporter:{{.Version}}-amd64"
+  dockerfile: Dockerfile
+  build_flag_templates:
+    - "--pull"
+    - "--platform=linux/amd64"
+- goos: linux
+  goarch: arm64
+  image_templates:
+  - "percona/mongodb_exporter:{{.Major}}.{{.Minor}}-arm64v8"
```

Comment:
Hi @charlie-haley, thanks for the contribution.
Do we need to add arch to the tag ?

so far build is broken for me under podman, but looks like that we would like to get arch inside minifests, not the tags.

percona/mongodb_exporter:0.31.2 could point to both platforms and cli would decide what image layers to pull based on a current platforms.

BTW, maybe we also could add GH repo to publish mongodb_exporter images as well, as we did it in :
https://github.com/percona/rds_exporter/blob/main/.goreleaser.yml#L43

Reply:
I managed to get it publishing to my own registry locally using `goreleaser release --skip-validate`, albeit with docker.

So the way it works is it publishes 2 container images with the architecture tag, then the `docker_manifests` section stitches them together so when you pull the image like `docker pull percona/mongodb_exporter:0.31.2` it will be a multi-arch image.

```yaml
docker_manifests:
  - name_template: percona/mongodb_exporter:{{.Major}}.{{.Minor}}
    image_templates:
    - percona/mongodb_exporter:{{.Major}}.{{.Minor}}-arm64v8
    - percona/mongodb_exporter:{{.Major}}.{{.Minor}}-amd64
```

example from our registry:
```
$ docker manifest inspect gcr.io/myregistry/percona/mongodb_exporter:0.31.2
{
   "schemaVersion": 2,
   "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
   "manifests": [
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 738,
         "digest": "sha256:6aadf5143ed49dea42bea857ed0a70abf87a84c371113e7f9b2deea12235a403",
         "platform": {
            "architecture": "amd64",
            "os": "linux"
         }
      },
      {
         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
         "size": 738,
         "digest": "sha256:c545df8c6715f16cbb83b2ef2c62597bfad90d9ff2ffaccef07ba6037725b5c7",
         "platform": {
            "architecture": "arm64",
            "os": "linux"
         }
      }
   ]
}
```

Reply:
I'll also update the PR to include a GHCR mirror now

Reply:
thanks for explanation, as I wasn't able to try it goreleaser config didn't helped me to understand it.

```
 > [builder 2/2] RUN apk add --no-cache ca-certificates:
------
Dockerfile:2
--------------------
   1 |     FROM alpine AS builder
   2 | >>> RUN apk add --no-cache ca-certificates
   3 |
   4 |     FROM scratch AS final
--------------------
error: failed to solve: process "/dev/.buildkit_qemu_emulator /bin/sh -c apk add --no-cache ca-certificates" did not complete successfully: failed to copy xattrs: failed to set xattr "security.selinux" on /tmp/buildkit-qemu-emulator1783888444/dev/.buildkit_qemu_emulator: operation not supported
```

SELinux is fun :)

Reply:
[Have you configured buildx + QEMU? ](https://docs.docker.com/buildx/working-with-buildx/#build-multi-platform-images)You may need to run:

```
docker run --privileged --rm tonistiigi/binfmt --install all
docker buildx create --use
```

Reply:
it was working before. I will figure it out later. Thanks for the changes, after @ShashankSinha252 review we can land it.